src/OpenTK/Math/Vector2.cs

```diff
@@ -780,11 +800,31 @@ public static Vector2 Transform(Matrix3 mat, Vector2 pos)
         /// <param name="mat">The desired transformation</param>
         /// <param name="pos">The position to transform</param>
         /// <param name="result">The transformed vector</param>
-        /// <returns>The transformed position</returns>
         public static void Transform(ref Matrix3 mat, ref Vector2 pos, out Vector2 result)
```

Comment:
This multiplication is still an undefined operation. For a given matrix with dimensions *m* by *n*, multiplication is only defined for another matrix (which we can consider a vector) of the size *n* by *p*. Therefore, it is undefined for multiplication between a Matrix3 (3x3) and a Vector2 (2x1).

Remove this method.